packages/client/components/EditingStatus/EditingStatusText.tsx

```diff
@@ -22,7 +21,7 @@ const useTimeFrom = (timestamp: string) => {
   useEffect(() => {
     timeoutRef.current = window.setTimeout(() => {
       setTimeFrom(makeTimeFrom())
-    }, getRefreshPeriod(timestamp))
```

Comment:
it's a performance thing. if you have a screen with 100 tasks, you probably don't want or need every card updating every second. updating every minute is fine for new cards, updating every day is OK for older cards IIRC

Reply:
-1 the correct thing to do is to leave `getRefreshPeriod` in place to keep the performance optimization, but to reset the time when the task was edited. It should work if you put `setTimeFrom(makeTimeFrom())` at the beginning of `useEffect` as this hook should be already executed when the timestamp changes. It should not have `timeFrom` in the dependencies though, because it's not dependent on that and would retrigger it continously.

Reply:
On toggling the timestamp it does not show the updated value immediately, that feels bug to me as a user.

@mattkrick 
> you probably don't want or need every card updating every second
 
I am not sure if I understand how it is working. Do you mean after changing it to 600ms, every card on the screen will keep updating time, for example, a card with `Created 4 minutes ago` changes to `Created 5 minutes ago` after one minute automatically? But I don't see that happening. Could you explain?

Reply:
We don't need to re-render components twice a second if they show a big time like "1 hour ago". Instead we should keep `getRefreshPeriod(timestamp)` which gives the update period based on the time shown. Additionally we should call `setTimeFrom(makeTimeFrom())` manually on edits so the time is reset immediately.

Reply:
I still don't get it😢, does it mean if I set it to 600ms all the cards will keep re-rendering every 600ms infinitely? But I don't see that happening–all other cards does not update every minute, only the card that I click updates. Sorry, if it is a noob question.